apply-dictionary-renderer-to-feature-layer/README.metadata.json

```diff
@@ -0,0 +1,30 @@
+{
+    "category": "Visualization",
+    "description": "Convert features into graphics to show them with mil2525d symbols.",
+    "formal_name": "ApplyDictionaryRendererToFeatureLayer",
+    "ignore": false,
+    "images": [
+        "apply-dictionary-renderer-to-feature-layer.png"
+    ],
+    "keywords": [
+        "military",
+        "symbol",
+        "DictionaryRenderer",
+        "DictionarySymbolStyle",
+        "FeatureLayer",
+        "Geodatabase"
+    ],
+    "language": "kotlin",
+    "redirect_from": "",
+    "relevant_apis": [
+        "DictionaryRenderer",
+        "DictionarySymbolStyle",
+        "FeatureLayer",
+        "Geodatabase"
+    ],
+    "snippets": [
+        "src/main/java/com/esri/arcgismaps/sample/applydictionaryrenderertofeaturelayer/DownloadActivity.kt",
+        "src/main/java/com/esri/arcgismaps/sample/applydictionaryrenderertofeaturelayer/MainActivity.kt"
+    ],
```

Comment:
I realize this is just an annoying quirk of the script but `MainAcitivty.kt` should be set as the first snippet.